src/main/kotlin/no/nav/tsm/syk_inn_api/pdf/PdfService.kt

```diff
@@ -13,7 +13,13 @@ class PdfService(
 ) {
     fun createSykmeldingPdf(sykmeldingId: UUID, hpr: String): Result<ByteArray?> {
         val sykmelding =
-            sykmeldingService.getSykmeldingById(sykmeldingId, hpr) ?: return Result.success(null)
+            sykmeldingService.getSykmeldingById(sykmeldingId) ?: return Result.success(null)
+
+        if (sykmelding.meta.sykmelder.hprNummer != hpr) {
+            return Result.failure(
+                IllegalStateException("Sykmelding $sykmeldingId not belong to hpr=$hpr")
```

Comment:
"Sykmelding $sykmeldingId does not belong to HPR-number $hpr"

Reply:
All your sykmelding are belong to us